### PR TITLE
Update readme for hybrid apps

### PR DESCRIPTION
**Pull request by pankaj443**
- Bump version
- Update docs for hybrid apps code snippet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,11 +96,34 @@ percyScreenshot(driver, name[, {
...
-
       `right` (int): Right coordinate of the ignore region.
     - Raises:Error: If top, bottom, left, or right is less than 0 or top is greater than or equal to bottom or left is greater than or equal to right.
     - valid: Ignore region should be within the boundaries of the screen.
+
+## Running with Hybrid Apps
+
+For a hybrid app, we need to switch to native context before taking screenshot.
+
+- Add a helper method similar to following:
```
Comment (ninadbstack) [inline README.md:109]:
```suggestion
- Add a helper method similar to following for say flutter based hybrid app:
```

File: README.md [hunk 2]
```diff
@@ -96,11 +96,34 @@ percyScreenshot(driver, name[, {
...
+- Call percyScreenshotFlutter helper function when you want to take screenshot.
+```js
+await percyScreenshotFlutter(driver, name[, {
+  fullscreen,
+  deviceName,
+  orientation,
+  statusBarHeight,
+  navigationBarHeight
+}])
+```
```
Comment (ninadbstack) [inline README.md:129]:
```suggestion
```
Note: For other hybrid apps the `await driver.switchContext('FLUTTER');` would change to context that it uses like say `WEBVIEW` etc. 
```

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@percy/appium-app",
   "description": "Appium client library for visual testing with Percy",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (ninadbstack) [inline package.json:4]: Dont do version bump yet, this can go with next release

Comment (pankaj443) [reply package.json:4]: this is for previous PR, wdio type fix

Comment (ninadbstack) [reply package.json:4]: Got it!

**Review by ninadbstack — APPROVED** [review]

**Pull request merged**